{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "from text_class_helper import TextClfHelper\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import seaborn as sns\n",
    "import helpers\n",
    "\n",
    "import collections\n",
    "import gensim\n",
    "\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn import svm\n",
    "from sklearn.preprocessing import label_binarize\n",
    "from sklearn.multiclass import OneVsRestClassifier\n",
    "from sklearn.pipeline import Pipeline\n",
    "from sklearn.feature_selection import SelectKBest, mutual_info_classif\n",
    "from sklearn.metrics import log_loss"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Importing csv...\n"
     ]
    }
   ],
   "source": [
    "text_clf_helper = TextClfHelper()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "txt_train = text_clf_helper.txt_train"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Human colorectal cancer cell lines are used widely to investigate tumor biology, experimental therapy, and biomarkers. However, to what extent these established cell lines represent and maintain the genetic diversity of primary cancers is uncertain. In this study, we profiled 70 colorectal cancer cell lines for mutations and DNA copy number by whole-exome sequencing and SNP microarray analyses, respectively. Gene expression was defined using RNA-Seq. Cell line data were compared with those published for primary colorectal cancers in The Cancer Genome Atlas. Notably, we found that exome mutation and DNA copy-number spectra in colorectal cancer cell lines closely resembled those seen in primary colorectal tumors. Similarities included the presence of two hypermutation phenotypes, as defined by signatures for defective DNA mismatch repair and DNA polymerase ε proofreading deficiency, along with concordant mutation profiles in the broadly altered WNT, MAPK, PI3K, TGFβ, and p53 pathways. Furthermore, we documented mutations enriched in genes involved in chromatin remodeling (ARID1A, CHD6, and SRCAP) and histone methylation or acetylation (ASH1L, EP300, EP400, MLL2, MLL3, PRDM2, and TRRAP). Chromosomal instability was prevalent in nonhypermutated cases, with similar patterns of chromosomal gains and losses. Although paired cell lines derived from the same tumor exhibited considerable mutation and DNA copy-number differences, in silico simulations suggest that these differences mainly reflected a preexisting heterogeneity in the tumor cells. In conclusion, our results establish that human colorectal cancer lines are representative of the main subtypes of primary tumors at the genomic level, further validating their utility as tools to investigate colorectal cancer biology and drug responses. Cancer Res; 74(12); 3238–47. ©2014 AACR.  Introduction  Colorectal cancer is a leading cause of cancer-related morbidity and mortality (1). Human colorectal cancer cell lines are an important, commonly used preclinical model system for studying this disease, and have provided essential insights into tumor molecular and cell biology. Cell lines are a fundamental tool used in the discovery of new antitumor compounds and for the discovery of drug sensitivity, resistance, and toxicity biomarkers, with molecular markers of response to conventional chemotherapies and targeted agents showing clinical utility in patients (2–7). Examples include the relationship between tumor microsatellite instability-high (MSI-H) status and lack of 5-fluorouracil (5FU) response (2), and mutations in KRAS, BRAF, and PIK3CA exon 20 and resistance to anti-EGFR antibody therapy (3). However, to what extent colorectal cancer cell lines represent and maintain the genetic diversity of primary cancers remains controversial.  More than the past two decades, major cancer genes and pathways central to colorectal cancer development have been delineated, including the WNT, MAPK, PI3K, TGFβ, and p53 pathways. Two broad molecular subtypes of colorectal cancer have emerged, characterized by MSI-H (∼15%) or chromosomal instability (CIN, ∼60%; refs. 8–10). MSI-H colorectal cancers occur predominantly in the proximal colon, and often show poor differentiation, mucinous histology, and increased peritumoral lymphocytic infiltration (11). These tumors exhibit hypermutation caused by defective DNA mismatch repair (MMR), tend to be near-diploid and to have a CpG island methylator phenotype (CIMP), and harbor mutations in a distinct set of driver genes, including BRAF, PTEN, and TGFBR2 (12–14). In contrast, chromosomally unstable tumors are more common in the distal colorectum and tend to develop along the classical genetic pathway of colorectal tumorigenesis, with mutations in APC, KRAS, SMAD4, and TP53 (15).  Recent cancer genome sequencing studies have revealed additional details of the genetic profiles of human colorectal cancer, highlighting their diversity. An initial whole-exome sequencing study on 11 microsatellite stable (MSS) colorectal cancers demonstrated that such tumors harbor ∼80 coding sequence mutations with a small number of commonly mutated driver genes and a large number of “private” mutations (16). Subsequently, The Cancer Genome Atlas (TCGA) Network reported comprehensive data on 223 unselected sporadic colorectal cancers (17). Hypermutation was identified in ∼15% of carcinomas, with three quarters of these displaying the expected MSI-high (MSI-H) phenotype associated with epigenetic silencing and/or mutation of MMR genes. However, one-quarter of hypermutated tumors did not show MSI-H and seemed to be associated with DNA polymerase ε (POLE) mutations, perhaps representing a distinct colorectal cancer subtype. Twenty-four genes were identified as significantly mutated in colorectal cancer, including several novel candidate genes such as ATM, ARID1A, TCF7L2, SOX9, and FAM123B. A number of recurrent DNA copy-number alterations were reported comprising potentially drug-targetable amplifications of ERBB2 and IGF2. A similar study on 74 pairs of primary human colon tumors reported highly concordant results, and in addition identified low frequency fusion transcripts involving R-spondin family members RSPO2 and RSPO3 (18).  Here, we report the first comprehensive whole-exome and DNA copy-number analyses of 70 of the most widely used colorectal cancer cell lines, and undertake a detailed comparison of genetic alterations with those reported for TCGA-analyzed primary cancers. We demonstrate that the spectra of mutations and DNA copy-number aberrations in colorectal cancer cell lines are representative of primary tumors, including hypermutation phenotypes and targeting of major signaling pathways. Our data further highlight novel aspects of colorectal cancer biology, including significant enrichment of mutated genes involved in chromatin remodeling and histone methylation or acetylation. Our results verify established colorectal cancer cell lines as a useful preclinical model system, and provide a comprehensive genomic data resource enabling informed choices when selecting cell lines for functional and pharmacogenomics research.  Materials and Methods  Colorectal cancer cell lines and TCGA-analyzed primary cancers  The 70 colorectal cancer cell lines studied were obtained from a range of sources, listed below, over a period spanning several years (Supplementary Table S1). C10, C106, C125, C135, C32, C70, C80, C84, and C99 were generated by the W.F. Bodmer laboratory, CACO2, COLO201, COLO205, COLO320-DM, DLD1, HCC2998, HCT116, HCT15, HCT8, HT29, LOVO, LS174T, LS180, LS411, LS513, NCI-H716, NCI-H747, RKO, SKCO-1, SNU-C1, SNU-C2B, SW1116, SW1417, SW1463, SW403, SW48, SW480, SW620, SW837, SW948, and T84 were obtained from the American Type Culture Collection, KM12 was obtained from DCTD, COLO678, and CX-1 were obtained from DSMZ, Gp2D, Gp5D, HT115, and HT55 were obtained from ECACC, CCK81, and CoCM-1 were obtained from HSRRB, VACO10, and VACO4S were provided by Dr. J.A. McBain (University of Wisconsin School of Medicine, Madison, WI), SNU-175, SNU-283, and SNU-C4 were obtained from KCLB, LIM1215, LIM1899, LIM2099, LIM2405, and LIM2551 were obtained from the Ludwig Institute for Cancer Research, SW1222 were provided by Prof. M. Herlyn (The Wistar Institute, Philadelphia, PA), HDC101, HDC54, HDC82, HDC87, and HDC90 were provided by Prof. M, Schwab (DKFZ, Germany), HCA46, HCA7, and HRA19 were provided by Dr. S.C. Kirkland (Royal Postgraduate Medical School, United Kingdom), and RW2982 and RW7213 were provided by Dr. P. Calabresi (Roger Williams General Hospital, Providence, RI). Cells were cultured with Dulbecco's Modified Eagle Medium and 10% FBS at 37°C and 5% CO2. Cell line authentication was performed using the Promega StemElite ID System at the Queensland Institute of Medical Research (QMIR, Queensland, Australia) DNA Sequencing and Fragment Analysis Facility (January 2013). Cells were tested for mycoplasma by the MycoAlert Mycoplasma Detection Kit (Lonza). Published exome-capture sequencing data on 223 patients with colorectal cancer were retrieved from the TCGA (19). SNP array data were available for 213 of these patients. Mutation data were filtered for exons and at splice sites (±3 bp).  Exome-capture sequencing  Exome-capture was performed using the TruSeq Exome Enrichment Kit (Illumina) and 100 bp paired-end read sequencing performed on an Illumina HiSeq 2000 at the Australian Genome Research Facility (AGRF). Variant detection was implemented using a modified GATK Best Practice protocol; variants were filtered against known germline variations and those detected in 114 in-house normal colorectal tissues (Supplementary Methods). Regions of known germline chromosomal segmental duplications were excluded (20).  Transcriptome sequencing  RNA-Seq analysis was performed at the AGRF on an Illumina HiSeq2000 to a depth of >100 million paired reads. Alignment to the human reference genome (hg19) was achieved using TopHat (21), and RPKM values calculated. Absence of gene expression was defined as a RPKM value of <1 (Supplementary Methods).  SNP microarray analysis  SNP assays were performed by the AGRF using Illumina Human610-Quad BeadChips (GSE55832) and processed using GenomeStudio (Illumina). SNPs showing germline alterations, based on 637 normal samples, were excluded. DNA copy-number segmentation was performed using OncoSNPv2.18 (22). Regions of significantly altered genome were identified using GISTIC2.0 (23).  Microsatellite instability analysis  MSI analysis was performed for the National Cancer Institute recommended microsatellite marker panel BAT25, BAT26, D2S123, D5S346, and D17S250 using fluorescently labeled primers on a 3130xl Genetic Analyzer (Applied Biosystems; ref. 24). MSI-H was diagnosed if instability was evident at 2 or more markers.  CIMP marker and MLH1 promoter methylation analysis  MethyLight real-time PCR was performed for the Weisenberger and colleagues CIMP marker panel (IGF2, SOCS1, RUNX3, CACNA1G, NGN1), MLH1, and the reference ALU (C-4; ref. 25). The percentage of methylated reference (PMR) was calculated for GENE:ALU ratio of template amount in a sample against GENE:ALU ratio of template amount in methylated reference DNA. Samples with a PMR greater than 10 for ≥3 CIMP markers were classified as CIMP-positive.  Statistical analysis  Statistical analyses were conducted using the R statistical computing software (http://www.R-project.org/). Differences between groups were assessed using Fisher exact test or χ2 test for categorical variables and the Student t test for continuous variables. Correlation between RNA-Seq and microarray gene expression data were assessed using Spearman rank correlation coefficient. Overlap of gene lists was assessed using a hypergeometric test. GISTIC2.0 analysis for significantly altered focal regions of chromosomal deletion or gain/amplification were adjusted for multiple testing, implementing a BH-FDR adjusted Q-value cut-off of 0.05. All statistical analyses were 2-sided and considered significant when P < 0.05.  Results  Exome mutation profiles in 70 colorectal cancer cell lines  Seventy commonly used colorectal cancer cell lines, comprising 63 unique lines and 7 duplicate lines, were analyzed for mutations in protein-coding exons and exon–intron boundaries by whole-exome capture sequencing (Supplementary Table S2). In all cases, >20-fold coverage of >80% of targeted exons was achieved. As matched normal tissue for these cell lines was not available, putative somatic mutations were identified by annotation against databases of known human germline variants. Regions of known germline chromosomal segmental duplications were excluded to reduce the possibility of false-positive variants caused by read mismapping. Mean pipeline sensitivity and specificity for nonsilent variants were shown to be 79.2% and 98.6% in an analysis of 10 primary colorectal cancers with and without paired normal tissue, with the majority (93.4%) of false-negative calls resulting from somatic mutations mimicking annotated germline variants (Supplementary Data). Accuracy of mutation calling was verified by Sanger resequencing for 12 selected genes (APC, CTNNB1, KRAS, BRAF, NRAS, PIK3CA, PTEN, SMAD2, SMAD3, SMAD4, FBXW7, and TP53) on 43 cell lines, with validation of 97.6% (165/169) of mutations (Supplementary Table S3).  For the 63 unique cell lines, the total number of putative somatic mutations ranged from 219 to 8657 (mean = 1498.3). Similar to the primary colorectal cancers reported by The TCGA (17), the prevalence of mutations varied substantially, ranging from 6.6 to 260.0 per 106 bases, with evidence for nonhypermutated and hypermutated groups (Fig. 1). Cell lines with a mutation prevalence of >25 per 106 bases were designated as hypermutated. Hypermutation was observed in 34.9% (22/63) of cell lines as compared with 15.7% (35/223) of the TCGA cancers (P < 0.001). 86.3% (19/22) of hypermutated cell lines exhibited MSI-H, similar to the proportion found in cancers (80.0%, 28/35, P = 0.725). MSI-H status in cell lines was associated with MLH1 hypermethylation, mutations in MMR genes and presence of CIMP (P < 0.009; Fig. 1). Consistent with the established association between MSI-H and proximal tumor location in colorectal cancer, all MSI-H cell lines with available site details originated from the proximal colon (Supplementary Table S1). There was no evidence of imbalance in site distribution between cell lines and cancers (P = 0.937).  Figure 1. Download figureOpen in new tabDownload powerpoint Figure 1. Mutation frequencies in 70 human colorectal cancer cell lines. Cell lines were separated into distinct hypermutated and nonhypermutated cases. MSI-H, microsatellite instability-high; CIMP, CpG island methylator phenotype; MLH1 meth., MLH1 promoter hypermethylation, MMR mut., DNA mismatch repair gene mutation; inset, mutations in MMR genes among the hypermutated samples, with cases in the same order as in the main graph. Mutation spectra in hypermutated colorectal cancer cell lines  Compared with nonhypermutated cell lines, MSI-H cell lines displayed a higher number of InDels (41.1-fold; P < 0.001) and SNVs (9.7-fold; P < 0.001) as observed in the TCGA cancers (InDels: 18.3-fold, SNVs: 10.8-fold; P < 0.001; Fig. 2A and B). As expected, InDels in MSI-H cases tended to occur at nucleotide-repeat (≥N5 or ≥NN3) sequences (MSI-H vs. nonhypermutated: cell lines 82.7% vs. 17.9%, cancers 73.4% vs. 38.4%; P < 0.001), a bias not observed for the SNVs (cell lines 2.9% vs. 3.7%, cancers 2.3% vs. 2.9%; Supplementary Table S2). The SNV spectrum differed significantly between MSI-H and nonhypermutated cases, with an increased proportion of A>G/T>C transitions (cell lines 1.5-fold, cancers 1.8-fold; P < 0.011) and decreased C>G/G>C transversions (cell lines −4.5-fold, cancers −5.2-fold; P < 0.001; Fig. 2C).  Figure 2. Download figureOpen in new tabDownload powerpoint Figure 2. Types of mutations for 70 human colorectal cancer cell lines and 223 TCGA-analyzed primary cancers. A–C, counts of SNVs and InDels (A and B), and proportions of nucleotide transitions and transversions (C). Paired cell lines originating from the same tumor are indicated by identical colored stars. MSI-H, microsatellite instability-high; NSHP, nucleotide-substitution hypermutator phenotype; POLE EDM, missense mutation in the POLE exonuclease domain. A second type of hypermutated tumor was observed among cell lines and the TCGA cancers. These cases were MSS and compared with nonhypermutated cases exhibited a nucleotide-substitution hypermutator phenotype (NSHP) characterized by a substantial increase of SNVs (cell lines 16.8-fold, cancers 56.8-fold; P < 0.005; Fig. 2A and B). The SNV spectrum in 2 of 3 NSHP cell lines (HT115, HCC2998) and 7 of 7 NSHP cancers further displayed increased proportions of C>A/G>T transversions (MSI-H vs. nonhypermutated: cell lines 1.5-fold, cancers 2.4-fold) and A>C/T>G transversions (cell line 3.8-fold, cancers 2.0-fold), as well as decreased C>G/G>C transversions (cell lines −13.6-fold, cancers −22.3-fold; Fig. 2C). Combining cell lines and cancers, this mutator phenotype was significantly associated with the presence of POLE mutations (NSHP: 90.0%, 9/10 vs. MSI-H: 29.8%, 14/47 vs. nonhypermutated: 1.3%, 3/229; P < 0.001). In addition, all nine NSHP cases with POLE mutation had at least 1 missense mutation in the POLE exonuclease domain (EDM), as compared with only 1 of 17 non-NSHP cases with POLE mutation (P < 0.001; Supplementary Table S4). 82.4% (14/17) of POLE mutated non-NSHP samples were MSI-H, and there was no evidence that the non-EDM POLE mutations in these cases modified the MSI-H phenotype with similar SNV frequencies and transition/transversion spectra compared with MSI-H cancers without POLE mutation (P > 0.05 for all comparisons).  A single NSHP cell line, HT55, exhibited a distinct bias to A>T/T>A transversions (49.0% vs. nonhypermutated mean 4.6%) and was wild type for POLE, but a similar case was not present among the TCGA cancers (Fig. 2C). This cell line harbored a heterozygous truncating mutation in the MMR gene PMS1, which may be related to this mutator phenotype, although Pms1-deficient mice have been reported to show no evidence of tumor predisposition or hypermutation (26).  Chromosomal and subchromosomal aberrations  DNA copy-number alterations in cell lines were profiled using Illumina Human 610-Quad BeadChips. As anticipated, DNA copy-number profiles were similar between cell lines and primary cancers (Fig. 3A). Hypermutated MSI-H and NSHP cases showed stable profiles with a modal chromosome copy-number of 2n. In contrast, nonhypermutated cases tended to exhibit unstable profiles with modal chromosome copy numbers ranging from 2n to 4n in cell lines and 2n to 6n in cancers. For nonhypermutated groups, the most common deleted chromosome arms were 8p, 17p (including TP53), and 18q (including SMAD4), and the most common gained regions were chromosome 7, 8q (including MYC), 13, and 20q. Some differences were apparent between cell lines and primary cancers, including more frequent chromosome 4 deletions in nonhypermutated and chromosome 7 gains and 18q deletions in hypermutated cell lines.  Figure 3. Download figureOpen in new tabDownload powerpoint Figure 3. Genome-wide DNA copy-number aberrations in 63 unique colorectal cancer cell lines and 213 TCGA primary cancers stratified into nonhypermutated and hypermutated cases. A, absolute DNA copy-number gains and losses relative to diploid status (2n) deduced using oncoSNP. B and C, minimal significant regions of chromosomal loss (blue) and gain/amplification (red) deduced using GISTIC2.0. Selected candidate genes in overlapping regions are indicated and total numbers of genes are shown in brackets. Focal regions significantly targeted by DNA copy-number alterations in both cell lines and cancers were deduced using GISTIC2.0 software (23). Eleven and 4 significant focal regions were found to overlap between nonhypermutated and hypermutated groups, respectively (Fig. 3B and C and Supplementary Table S5). For nonhypermutated cases, overlapping regions included gain/amplification of MYC, a known key mediator of colorectal tumorigenesis (27), and deletion of the candidate cancer genes PARK2 and WRN, detected in 29.5% and 36.3% of cell lines and 21.9% and 31.6% of primary cancers, respectively. The 4 recurrent regions identified in hypermutated cases—which were also present in the nonhypermutated group—were all known fragile sites (including FHIT, A2BP1, WWOX, or MACROD2), the functional relevance of which is uncertain.  Cancer genes and pathways  Gene mutation profiles, excluding silent mutations, were compared between colorectal cancer cell lines and primary cancers. Only genes with well-defined expression in cell lines [reads per kilobase per million reads mapped (RPKM) >1 in at least 3/13 colorectal cancer cell lines analyzed by RNA-Seq] were considered (n = 20,702 genes; Supplementary Table S6). NSHP cases were excluded from this comparison because of the small sample size.  Mutation landscapes of cell lines markedly resembled those of primary cancers. In both cohorts, nonhypermutated cases displayed a small number of distinct mutation peaks, with APC, TP53, and KRAS being the most common targets, whereas hypermutated MSI-H cases showed frequent mutations in multiple genes, with the anticipated bias to genes comprising nucleotide repeats (Supplementary Tables S7). Significant overlap was observed for the top 5% of mutated genes (based on the proportion of affected samples), with 54 and 62 genes intersecting for nonhypermutated and MSI-H groups, respectively (P < 0.001; Supplementary Tables S8). Overlapping genes included the expected members of the WNT, MAPK, PI3K, TGFβ, and p53 pathways. For nonhypermutated cases, these comprised APC, CTNNB1, FBXW7, KRAS, BRAF, PIK3CA, SMAD4, and TP53, as well as the candidate cancer genes, AXIN2, BCL9L, FAT1, SOX9, ERBB3, PIK3C2B, TIAM1, and ATM. For MSI-H cases these encompassed APC, FBXW7, PIK3CA, and TGFBR2, along with the candidates CREBBP, TCF7L2, RNF43, and ACVR2A. Mutation distributions across colorectal cancer-associated signaling pathways were also similar between nonhypermutated cell lines and cancers, with the same pathway members tending to show the highest mutation frequencies (Fig. 4). Greater variability in mutation frequencies between pathway members was observed for MSI-H cases, at least partly related to the higher mutation background and smaller sample size. A notable difference for MSI-H cases was a differential prevalence of CTNNB1 mutations, which were frequent in cell lines (47%) but not reported in primary cancers (P < 0.001).  Figure 4. Download figureOpen in new tabDownload powerpoint Figure 4. Mutation frequencies for WNT, MAPK, PI3K, TGFβ, and p53 pathway members as well as chromatin regulators identified among the top 5% of mutated genes overlapping between colorectal cancer cell lines and TCGA-analyzed primary cancers for nonhypermutated or hypermutated MSI-H samples. In addition to the established colorectal cancer-associated pathways, significant enrichment was observed for mutations in chromatin-state regulators (P < 0.001; gene ontology enrichment analysis). These comprised ASH1L, CHD6, PRDM2, and MLL3 in nonhypermutated and ARID1A, EP300, EP400, MLL2, CHD6, PRDM2, TRRAP, and SRCAP in MSI-H cases (Supplementary Table S8). Approximately 49% and 19% of nonhypermutated and 100% and 93% of MSI-H cell lines and primary cancers carried mutations across these candidates, respectively.  Integrating mutation and DNA copy-number data across cell lines and primary cancers showed the anticipated tumor suppressor signatures for APC, TP53, SMAD4, and SOX9 with a significant overrepresentation of 2 mutational hits (2+ mutations or 1 mutation and loss of heterozygosity; P < 0.023). There was further an association of mutation in BRAF, ERBB3, PIK3CA, and KRAS with copy-number gain of ≥5 at the respective chromosomal regions (P < 0.018). Trends to mutual exclusivity between pathway member mutations were observed for KRAS and BRAF, TP53, and ATM (P < 0.005).  Mutation and DNA copy-number differences in paired colorectal cancer cell lines  Included in our colorectal cancer cell line panel were 5 pairs/triplets originally derived from the same tumor (COLO201/COLO205, CX-1/HT29, Gp2D/Gp5D, LS174T/LS180, and DLD1/HCT8/HCT15), and 1 pair derived from a primary tumor and subsequent lymph node metastasis (SW480/SW620). LS174T and LS180 were established by trypsin treatment and scraping of primary cultures from the same tumor, respectively (28), and have been shown to differ with respect to E-cadherin expression and cell adhesion, with LS174T displaying complete loss of E-cadherin protein (2, 29).  Assessment of the overlap between mutations detected in paired cell lines identified substantial discrepancies, with 63 and 356 mutational differences in the nonhypermutated pairs COLO201/COLO205 and CX-1/HT29, and 2,763, 480, 6,503, 5,377, and 6,369 mutational differences in the MSI-H pairs Gp2D/Gp5D, LS174T/LS180, DLD1/HCT8, DLD1/HCT15, and HCT8/HCT15 (Fig. 5). Eight hundred and forty-nine mutations differed in the nonhypermutated primary/metastasis pair SW480/SW620. Nonsilent and silent mutations contributed in similar proportions to these discrepancies (54.4% vs. 56.6%), suggesting no selection for these differential alterations.  Figure 5. Download figureOpen in new tabDownload powerpoint Figure 5. Overlap of mutations and DNA copy-number states between paired colorectal cancer cell lines. COLO201/COLO205, CX-1/HT29, Gp2D/Gp5D, LS174T/LS180, and DLD1/HCT8/HCT15 were derived from the same tumor, SW480/SW620 from a primary tumor and subsequent lymph node metastasis. DNA copy-number profiles showed multiple differences for nonhypermutated pairs, with 41.1% and 53.8% of the genome varying for COLO201/COLO205 and CX-1/HT29 (Fig. 5). In contrast, discrepancies were limited in MSI-H pairs with 0.2%, 0.4%, 6.6%, 2.9%, and 4.3% of the genome differing between Gp2D/Gp5D, LS174T/LS180, DLD1/HCT8, DLD1/HCT15, and HCT8/HCT15, respectively. 47.5% of the genome differed in the nonhypermutated primary/metastasis pair SW480/SW620.  Notably, mutation differences between paired cell lines did not obscure known driver genes. For the established colorectal cancer genes APC, TP53, SMAD4, PIK3CA, KRAS, and BRAF, 29 of 33 (87.9%) nonsilent mutations were concordant between cell line pairs.  Discussion  In this study we show that the mutation and DNA copy-number landscapes determined for 70 colorectal cancer cell lines closely resemble those of primary tumors, underscoring the utility of cell lines as an appropriate model system for this malignancy. The 3 molecular subtypes of colorectal cancer recently defined by the TCGA, nonhypermutated, hypermutated with microsatellite instability, and hypermutated without microsatellite instability were faithfully captured in the cell line panel. As expected, MSI-H cell lines exhibited hypermethylation of the MLH1 promoter and/or mutations in MMR genes, whereas hypermutated lines without microsatellite instability were instead characterized by mutations in the exonuclease domain of POLE. Consistent with defective DNA POLE proofreading function, the latter tumors showed a nucleotide substitution hypermutator phenotype with a bias to C>A/G>T and A>C/T>G transversions. Germline missense mutations in the POLE exonuclease domain have recently been associated with familial predisposition to colorectal cancer (30), and elevated tumor predisposition and base-substitution mutations observed in Pole exonuclease-mutant mice (31). Similarly, somatic missense mutations in the POLE exonuclease domain have been reported in 7% of endometrial cancers, with good evidence of associated hypermutation (32). Although differences in prognosis and chemotherapy response have been extensively documented for nonhypermutated versus MSI-H tumors (2–4), clinical characteristics of POLE-mutant NSHP tumors are unknown. Our identification of 2 cell lines representative of this colorectal cancer subtype will facilitate an investigation of the specific aspects of the biology of these tumors, particularly in relation to identifying therapeutics that may exploit their unique genomic instability.  Consistent with previous lower-resolution data (33), DNA copy-number profiles were similar between colorectal cancer cell lines and primary tumors, with nonhypermutated cases tending to exhibit chromosomal instability, whereas both MSI-H and NSHP cases had overall stable copy-number profiles. Patterns of whole and partial chromosome gains and losses were largely concordant. Besides recurrent gain/amplification of the established colorectal cancer gene MYC, recurrent regions of deletion in nonhypermutated cases included the candidate cancer genes PARK2 and WRN. WRN has important roles in homologous recombination repair, MUTYH-mediated repair of oxidative DNA damage, and telomeric recombination (34–36), and WRN germline mutations are associated with chromosomal instability and cancer predisposition (37). PARK2 deletion has been previously reported in sporadic colorectal cancer, and Park2 deficiency shown to accelerate intestinal adenoma development in Apc mutant mice (38).  The mutation landscapes in nonhypermutated and hypermutated MSI-H cases showed close resemblance in cell lines and primary tumors, with the expected alterations in the WNT, MAPK, PI3K, p53, and TGFβ pathways. Besides these main colorectal cancer–associated pathways, multiple chromatin-state regulators were enriched among the top 5% of mutated genes, including proteins involved in chromatin remodeling (ARID1A, CHD6, and SRCAP) and histone methylation or acetylation (ASH1L, EP300, EP400, MLL2, MLL3, PRDM2, and TRRAP). Overall, ∼24% of nonhypermutated and ∼96% of MSI-H cases harbored mutations in these genes. Trends for mutations to cluster in chromatin regulators have been reported in multiple other cancer types, including liver cancers (39), gastric adenocarcinoma (40), and transitional cell carcinoma of the bladder (41), suggesting potential pathogenicity of these alterations.  Paired cell lines originating from the same tumor showed considerable differences for both mutations and DNA copy-number profiles. Nonhypermutated pairs differed for up to 356 mutations and ∼54% of genome copy number, whereas MSI-H pairs differed for up to 6,503 mutations and ∼6.6% of genome copy number. The two main possible reasons for these differences are preexisting heterogeneity between the cells from the original tumor grown out to establish these paired lines, or acquisition of alterations as a result of long-term culture. Assuming a normal mutation rate of 10−8 (per bp per cell generation) for nonhypermutated and 100-fold elevated rate for hypermutated pairs and absence of selection (42), cell lines established from tumor cells separated by an average of 328 and 66 replications would be expected to exhibit the observed mutational differences with a >99% probability, respectively (Supplementary Data and Table S9). In contrast, mutations acquired during serial passaging in culture are not anticipated to reach a detectable 10% level before 9,034 and 4,266 replications, respectively (Supplementary Data and Fig. 6). Although serial cell passaging may have been common-place at the time when these paired cell lines were established >20 years ago, stock-keeping practices to limit the number of replications were soon introduced. Preexisting mutation heterogeneity in the original tumor therefore seems highly likely to account for the majority of the detected differences. Numbers of replications may be expected to be larger between cells from a primary tumor and subsequent metastasis, and accordingly our corresponding nonhypermutated cell line pair showed a ∼4-fold higher number of mutational differences than the other nonhypermutated pairs. Importantly, mutational differences across paired cell lines did not obscure known driver genes, with ∼88% of nonsilent mutations in established colorectal cancer genes concordant between pairs.  Figure 6. Download figureOpen in new tabDownload powerpoint Figure 6. Simulation of the acquisition of mutations in cell culture. The process of serial passage was modeled with cell numbers repeatedly increasing from 1 × 105 to 2 × 106 cells. Proportions of cells containing the most frequent mutation by passage number for five simulations, using fixed mutation probabilities of 10−8 per base per cell replication for nonhypermutated (A) and 10−6 for hypermutated (B) MSI-H tumors. The black diagonal trendline is the least squares fit on the log–log scale and the small vertical lines are the 99% prediction intervals. The horizontal red line corresponds to a sequencing mutation detection threshold of 10%. Despite the high level of similarity between colorectal cancer cell lines and primary tumors, there were a number of differences. These included overall higher mutation and DNA copy-number frequencies, as well as a greater proportion of detected InDels in cell lines. These discrepancies may be related to differences in exome-capture and sequencing platforms, bioinformatics pipelines, the presence of contaminating normal tissue in primary cancers, and accuracy of assigning somatic alterations in cell lines. Cell lines further showed a higher proportion of hypermutated cases, and exhibited differences in the prevalence of aberrations for certain genes and genomic regions such as CTNNB1 mutations in MSI-H cases. These latter findings may be a reflection of preferential growing out of cell lines from primary tumors (or their respective subclones) that contain these aberrations, a contention supported by the observation that only ∼10% to 15% of colorectal cancers give rise to cell lines (43). Another possibility is that some of the mutations have been acquired and selected for in tissue culture.  A caveat to our analysis of protein-coding genes is that we could not report on untranslated exonic regions (UTR), as the latter were inconsistently covered by our study and the TCGA. UTR mutations can impact on RNA splicing, stability, or translation as previously highlighted for MSI-H colorectal cancer (44). In the comparison of gene mutation profiles, we chose to exclude silent mutations (other than those affecting splice sites), although some of these may similarly have functional consequences (45).  In conclusion, our comparative analysis of the genomic landscapes of human colorectal cancer cell lines and TCGA-analyzed primary cancers identified cell lines representative of the three main mutational colorectal cancer subtypes. Within these molecular subtypes, although some differences were evident, cell lines showed globally similar genetic alterations to primary cancers, including genome-wide mutation, DNA copy number, and driver gene mutation profiles. Accordingly, gene expression profiles of colorectal cancer cell lines have previously been shown to broadly represent those of primary tumors (46). Our genomic data significantly expand on cancer cell line characterization efforts by the major cancer genome centers, such as the Cancer Cell Line Encyclopedia project, which currently reports mutation data for 1,500 selected genes on 62 colorectal cancer cell lines (5). Our data will help to inform investigations of the molecular basis of colorectal cancer pathogenesis, inherent and acquired drug resistance, and exploration of novel treatment modalities for this malignancy. To characterize somatic alterations in colorectal carcinoma (CRC), we conducted genome-scale analysis of 276 samples, analyzing exome sequence, DNA copy number, promoter methylation, mRNA and microRNA expression. A subset (97) underwent low-depth-of-coverage whole-genome sequencing. 16% of CRC have hypermutation, three quarters of which have the expected high microsatellite instability (MSI), usually with hypermethylation and MLH1 silencing, but one quarter has somatic mismatch repair gene mutations. Excluding hypermutated cancers, colon and rectum cancers have remarkably similar patterns of genomic alteration. Twenty-four genes are significantly mutated. In addition to the expected APC, TP53, SMAD4, PIK3CA and KRAS mutations, we found frequent mutations in ARID1A, SOX9, and FAM123B/WTX. Recurrent copy number alterations include potentially drug-targetable amplifications of ERBB2 and newly discovered amplification of IGF2. Recurrent chromosomal translocations include fusion of NAV2 and WNT pathway member TCF7L1. Integrative analyses suggest new markers for aggressive CRC and important role for MYC-directed transcriptional activation and repression.  Go to: Background The Cancer Genome Atlas (TCGA) project plans to profile genomic changes in 20 different cancer types and has published results on two cancer types1,2. We now present results from multidimensional analyses of human colorectal cancer (CRC).  CRC is an important contributor to cancer mortality and morbidity. The distinction between colon and rectum is largely anatomical, but it impacts both surgical and radiotherapeutic management and it may impact prognosis. Most investigators divide CRC biologically into those with microsatellite instability (MSI) (located primarily in the right colon and frequently associated with the CpG island methylator phenotype (CIMP) and hyper-mutation) and those that are microsatellite-stable (MSS) but chromosomally unstable (CIN).  A rich history of investigations (for a review see3) has revealed several critical genes and pathways important to the initiation and progression of CRC3. These include the WNT, RAS-MAPK, PI3K, TGF-β, P53 and DNA mismatch repair pathways. Large-scale sequencing analyses4–6 have identified numerous recurrently mutated genes and a recurrent chromosomal translocation. Despite that background, we have not had a fully integrated view of the genetic and genomic changes and their significance for colorectal tumorigenesis. Further insight into those changes may enable deeper understanding of the pathophysiology of CRC and may identify potential therapeutic targets.  Go to: Results Tumor/normal pairs were analyzed by different platforms. The specific numbers of samples analyzed by each platform are shown in Supplementary Table 1.  Exome sequence analysis  To define the mutational spectrum, we performed exome capture DNA sequencing on 224 tumor/normal pairs (Supplementary Table 2 lists all mutations). Sequencing achieved >20X coverage of at least 80% of targeted exons. The somatic mutation rates varied considerably among the samples. Some had mutation rates <1/106 bases, whereas a few had mutations rates >100/106. We separated those cases (84%) with a mutation rate <8.24/106 (median number of non-synonymous mutations: 58) and those with mutations rates >12/106 (median number of mutations: 728), which we designated as hypermutated (Figure 1).  Figure 1 Figure 1 Mutation frequencies in human CRC To assess the basis for the strikingly different mutation rates, we evaluated microsatellite instability (MSI)7 and mutations in the DNA mismatch repair pathway8–10 genes MLH1, MLH3, MSH2, MSH3, MSH6 and PMS2. Among the 30 hypermutated tumors with a complete data set, 23 (77%) had high levels of MSI (MSI-H). Included were 19 with MLH1 methylation, 17 of which had high CpG island methylation phenotype (CIMP). By comparison, the remaining seven hypermutated tumors, including the six with the highest mutation rates, lacked MSI-H, CIMP or MLH1 methylation but usually had somatic mutations in one or more mismatch repair genes or Polε aberrations rarely seen in the non-hypermutated tumors (Figure 1).  Gene mutations  Overall, we identified 32 somatic recurrently mutated genes (defined by MutSig11 and manual curation) in the hypermutated and non-hypermutated cancers (Figure 1B). After removal of non-expressed genes, there were 15 and 17, respectively, in the hypermutated and non-hypermutated cancers (Figure 1B, see Supplementary Table 3 for complete list). Among the non-hypermutated tumors, the eight most frequently mutated genes were APC, TP53, KRAS, PIK3CA, FBXW7, SMAD4, TCF7L2 and NRAS. As expected, the mutated KRAS and NRAS genes usually had oncogenic codon 12/13 or 61 mutations, whereas the remaining genes had inactivating mutations. CTNNB1, SMAD2, FAM123B and SOX9 were also mutated frequently. FAM123B (WTX) is an X-linked negative regulator of WNT signaling12, and virtually all its mutations were loss-of-function. Mutations in SOX9, a gene important in cell differentiation in the intestinal stem cell niche13,14, have not been associated previously with human cancer, but all nine mutated alleles in the non-hypermutated CRCs were frameshift or nonsense mutations. Tumor suppressors ATM and ARID1A also had a disproportionately high number of frameshift or nonsense mutations. ARID1A mutations have recently been reported in CRC and many other cancers15,16.  In the hypermutated tumors, ACVR2A, APC, TGFBR2, MSH3, MSH6, SLC9A9 and TCF7L2 were frequent targets of mutation (Figure 1B), along with mostly BRAF V600E mutations. However, two genes that were frequently mutated in the non-hypermutated cancers were significantly less frequently mutated in hypermutated tumors: TP53 (60 vs 20%, p < 0.0001), and APC (81% vs 51%, p = 0.0023, both Fisher’s exact test). Other genes, including TGFBR2, were recurrently mutated in the hypermutated cancers, but not in the non-hypermutated samples. These findings suggest that hypermutated and non-hypermutated tumors progress through different sequences of genetic events.  As expected, hypermutated tumors with MLH1 silencing and MSI-H exhibited additional differences in mutational profile. When we specifically examined 28 genes with long mononucleotide repeats in their coding sequences, we found that the rate of frameshift mutation was 3.6-fold higher than the rate of such mutation in hypermutated tumors without MLH1 silencing, and 50-fold higher than in non-hypermethylated tumors (Supplementary Table 2).  Classification of tumors based on mutation rate and methylation pattern  As mentioned above, patients with colon and rectal tumors are managed differently17, and epidemiology also shows differences between the two17. An initial integrative analysis of MSI status, somatic copy number alterations (SCNAs), CIMP status and gene expression profiles of 132 colonic and 62 rectal tumors enabled us to examine possible biological differences between tumors in the two locations. Among the non-hypermutated tumors, however, the overall patterns of changes in copy number, CIMP, mRNA and miRNA were indistinguishable between colon and rectal carcinomas (Figure 2). Based on that result, we merged the two for all subsequent analyses.  Figure 2 Figure 2 Integrative analysis of genomic changes in 195 CRC tumors Unsupervised clustering of the promoter DNA methylation profiles of 236 colorectal tumors revealed four subgroups (Supplementary Methods; Supplementary Figure 1). Two of the clusters contained tumors with elevated rates of methylation and were classified as CIMP-high (CIMP-H) and CIMP-low (CIMP-L), as previously described18. The two non-CIMP clusters were predominantly from tumors that were non-hypermutated and derived from different anatomic locations. mRNA expression profiles separated the colorectal tumors into three distinct clusters (Supplementary Figure 2). One significantly overlapped with CIMP-H tumors (p=3×10−12) and was enriched with hypermutated tumors; the other two clusters did not correspond with any group in the methylation data. Analysis of miRNA expression by unsupervised clustering (Supplementary Figure 3) identified no clear distinctions between rectal cancers and non-hypermethylated colon cancers.  Chromosomal and sub-chromosomal changes  257 tumors were profiled for somatic copy-number alterations (SCNAs) with Affymetrix SNP 6.0 arrays. Of those tumors, 97 were also analyzed by low depth-of-coverage (low-pass) whole-genome sequencing (WGS). As expected, the hypermutated tumors had far fewer SCNAs (Figure 2). No difference was found between MSI and MSS hypermutated tumors (Supplementary Figure 4). We used the GISTIC algorithm19 to identify likely gene targets of focal alterations. There were several previously well-defined arm-level changes, including gains of 1q, 7p/q, 8p/q, 12q, 13q, 19q, and 20p/q6 (Supplementary Figure 4; Supplementary Table 4). Significantly deleted chromosome arms were 18p/q (including SMAD4) in 66% of the tumors and 17p/q (including TP53) in 56%. Also significantly deleted genes were 1p, 4q, 5q, 8p, 14q, 15q, 20p, and 22q.  We identified 28 recurrent deletion peaks (Supplementary Table 4; Supplementary Figure 4), including genes like FHIT, A2BP1 and WWOX with large genomic footprints located in potentially fragile sites of the genome, in near-diploid hypermutated tumors. Other focal deletions involved tumor suppressor genes such as SMAD4, APC, PTEN and SMAD3. A significant focal deletion of 10p25.2 spanned four genes, including TCF7L2, which was also frequently mutated in our dataset. A gene fusion between adjacent genes VTI1A and TCF7L2 through an interstitial deletion was found in 3% of CRCs and is required for survival of CRC cells bearing the translocation4.  There were 17 regions of significant focal amplification (Supplementary Table 4). Some of them were superimposed on broad gains of chromosome arms. Included were a peak at 13q12.13 near the peptidase gene USP12 and ~500kB distal to the CRC candidate oncogene CDK8; an adjacent peak at 13q12; a peak containing KLF5 at 13q22.1; and a peak at 20q13.12 adjacent to HNF4A. Peaks on chromosome 8 included 8p12 (which contains the histone methyl-transferase WHSC1L1, adjacent to FGFR1) and 8q24 (which contains MYC). An amplicon at 17q21.1, found in 4% of the tumors, contains seven genes, including the tyrosine kinase ERBB2. ERBB2 amplifications have been described in colon, breast and gastric/esophageal tumors, and breast and gastric cancers bearing these amplifications have been treated effectively with the anti-ERBB2 antibody trastuzumab20–22.  One of the most common focal amplifications, found in 7% of the tumors, is gain of a 100–150 kb region of chromosome arm 11p15.5. It contains the genes encoding insulin (INS), insulin-like growth factor 2 (IGF2), and tyrosine hydroxylase (TH), as well as miR-483, which is embedded within IGF2 (Figure 3a). We found elevated expression of IGF2 and miR-483 but not of INS and TH (Figure 3b–c). Immediately adjacent to the amplified region is ASCL2, a transcription factor active in specifying intestinal stem cell fate23. Although ASCL2 has been implicated as a target of amplification in CRC23–25, it was consistently outside the region of amplification and its expression was not correlated with copy-number changes. These observations suggest that IGF2 and miR-483 are candidate functional targets of 11p15.5 amplification. IGF2 overexpression through loss of imprinting has been implicated in the promotion of CRC26,27. MiR-483 may also play a role in CRC pathogenesis28.  Figure 3 Figure 3 Copy number changes and structural aberrations in CRC A subset of tumors (15%) without IGF2 amplification also had dramatically higher levels (as much as 100X) of IGF2 gene expression, an effect not attributable to methylation changes at the IGF2 promoter. To assess the context of IGF2 amplification/overexpression, we systematically looked for mutually exclusive genomic events using the MEMo method29. We found a pattern of near exclusivity (corrected p < 0.01) of IGF2 overexpression with genomic events known to activate the PI3-K pathway (mutations of PIK3CA/PIK3R1 or deletion/mutation of PTEN, Figure 3c, and Supplementary Table 5). The IRS2 gene, whose product links IGF1R, the receptor for IGF2, with PI3-K, is on chromosome 13, which is frequently gained in colorectal cancer. Those cases with the highest IRS2 expression were mutually exclusive of the cases with IGF2 overexpression (p= 0.04) and also lacked mutations in the PI3-K pathway (p= 0.0001)(Figure 3c). Those results strongly suggest that the IGF2/IGF1R/IRS2 axis signals to PI3-K in CRC and imply that therapeutic targeting of the pathway could act to block PI3-K activity in this subset of patients.  Translocations  To identify novel chromosomal translocations, we performed low-pass, paired-end, whole-genome sequencing on 97 tumors with matched normals. In each case we achieved sequence coverage of ~3–4X and a corresponding physical coverage of 7.5–10X. Despite the low genome coverage, we detected 250 candidate inter-chromosomal translocation events (range 0–10/tumor). Among those events, 212 had one or both breakpoints in an intergenic region, whereas the remaining 38 juxtaposed coding regions of two genes in putative fusion events, of which 18 were predicted to code for in-frame events (Supplementary Table 6). We found three separate cases in which the first two exons of the NAV2 gene on chromosome 11 are joined with the 3’ coding portion of TCF7L1 on chromosome 2 (Supplementary Figure 5). TCF7L1 encodes TCF3, a member of the TCF/LEF class of transcription factors that heterodimerize with nuclear β-catenin to enable β-catenin-mediated transcriptional regulation. Intriguingly, in all three cases, the predicted structure of the NAV2-TCF7L1 fusion protein lacks the TCF3 β-catenin binding domain. This translocation is similar to another recurrent translocation identified in CRC, a fusion in which the amino terminus of VTI1A is joined to TCF4 that is encoded by TCF7L2, deleted or mutated in 12% of non-hypermutated tumors and a homolog of TCF7L14. We also observed 21 cases of translocation involving TTC28 located on Chromosome 22 (Supplementary Table 6). In all cases the fusions predict inactivation of TTC28, which has been identified as a target of p53 and an inhibitor of tumor cell growth30. Eleven of the 19 (58%) gene-gene translocations are validated by either obtaining PCR products and in some case sequencing the junction fragments (Supplementary Figure 5).  Altered pathways in CRC  Integrated analysis of mutations, copy-number, and mRNA expression changes in 195 tumors with complete data enriched our understanding of how some well-defined pathways are deregulated. We grouped samples by hypermutation status and identified recurrent alterations in the WNT, MAPK, PI3K, TGF-β and p53 pathways (Figure 4, Supplementary Figure 6, Supplementary Table 1).  Figure 4 Figure 4 Diversity and frequency of genetic changes leading to deregulation of signaling pathways in CRC We found that the WNT signaling pathway was altered in 93% of all tumors, including biallelic inactivation of APC (Supplementary Table 7) or activating mutations of CTNNB1 in ~80% of cases. There were also mutations in SOX9 and mutations and deletions in TCF7L2, as well as the DKK family members and AXIN2, FBXW7 (Supplementary Figure 7), ARID1A and FAM123B/WTX (the latter a negative regulator of WNT/β-catenin signaling12 found mutated in Wilm’s tumor31). A few mutations in FAM123B/WTX have been described in colorectal cancer32. SOX9 has been suggested to play a role in cancer, but no mutations have previously been described. The WNT receptor Frizzled (FZD10) was overexpressed in ~17% of samples, in some instances at levels 100X normal. Altogether, we found 16 different altered WNT pathway genes, confirming the importance of that pathway in CRC. Interestingly, many of those alterations were found in tumors that harbor APC mutations, suggesting that multiple lesions affecting the WNT signaling pathway confer selective advantage.  Genetic alterations in the PI3K and RAS-MAPK pathways are common in CRC. In addition to IGF2 and IRS2 overexpression, we found mutually exclusive mutations in PIK3R1 and PIK3CA as well as deletions in PTEN in 2%, 15% and 4% of non-hypermutated tumors, respectively. We found that 55% of non-hypermutated tumors have alterations in KRAS, NRAS or BRAF, with a significant pattern of mutual exclusivity (Supplementary Figure 6, Supplementary Table 1). We also evaluated mutations in the ERBB family of receptors because of the translational relevance of such mutations. Mutations or amplifications in one of the four genes are present in 22/165 (13%) non-hypermutated and 16/30 (53%) hypermutated cases. Some of the mutations are listed in the COSMIC database33, suggesting a functional role. Intriguingly, recurrent V842I ERBB2 and V104M ERBB3 mutations were found in four and two non-hypermutated cases, respectively. Mutations and focal amplifications of ERBB2 (Supplementary Figure 6) should be evaluated as predictors of response to agents that target those receptors. We observed co-occurrence of alterations involving the RAS and PI3K pathways in a third of tumors (Figure 4; Fisher’s exact test p = 0.039). These results suggest that simultaneous inhibition of the RAS and PI3K pathways may be required to achieve therapeutic benefit.  The TGF-β signaling pathway is known to be deregulated in colorectal and other cancers34. We found genomic alterations in TGFBR1, TGFBR2, ACVR2A, ACVR1B, SMAD2, SMAD3 and SMAD4 in 27% of the non-hypermutated and 87% of the hypermutated tumors. We also evaluated the p53 pathway, finding alterations in TP53 in 59% of non-hypermutated cases (mostly biallelic, Supplementary Table 8) and alterations in ATM, a kinase that phosphorylates and activates p53 following DNA damage, in 7%. Alterations in those two genes showed a trend towards mutual exclusivity (p = 0.016) (Figure 4, Supplementary Figure 6, Supplementary Table 1).  We integrated copy number, gene expression, methylation and pathway data using the PARADIGM software platform35. The analysis revealed a number of novel characteristics of CRC (Figure 5A). For example, despite the diversity in anatomical origin or mutation levels, nearly 100% of these tumors have changes in MYC transcriptional targets, both those promoted by and those inhibited by MYC. These findings are consistent with patterns deduced from genetic alterations (Figure 4) and suggest an important role for MYC in the CRC. The analysis also identified several gene networks altered across all tumor samples and those with differential alterations in hypermutated vs. non-hypermutated samples (Supplementary Table 7, Supplemental Data on the TCGA publication webpage).  Figure 5 Figure 5 Integrative analyses of multiple data sets Since most of the tumors used in this study were derived from prospective collection, survival data are not available. However, the tumors can be classified as aggressive or non-aggressive on the basis of tumor stage, lymph node status, distant metastasis and vascular invasion at the time of surgery. We found numerous molecular signatures associated with tumor aggressiveness, a subset of which is shown in Figure 5B. They include specific focal amplifications and deletions, and altered gene expression levels, including those of SCN5A36, a reported regulator of colon cancer invasion (full list: Supplementary Tables 10–11). Association with tumor aggressiveness is also observed in altered expression of miRNAs and specific somatic mutations (APC, TP53, PIK3CA, BRAF, and FBXW7; Supplementary Figure 8B). Mutations in FBXW7 (38 cases) and distant metastasis (32 cases) never co-occurred (p = 0.0019). Interestingly, a number of genomic regions have multiple molecular associations with tumor aggressiveness that manifest as “clinically-related genomic hotspots”. Examples of this are the region 20q13.12, which includes a focal amplification and multiple genes correlating with tumor aggression, and the region 22q12.3, containing APOL637 (Supplementary Figures 8–9).  Go to: Discussion This comprehensive integrative analysis of 224 colorectal tumor/normal pairs provides a number of insights into the biology of CRC and identifies potential therapeutic targets. To identify possible biological differences in colon and rectum tumors we found, in the non-hypermutated tumors, irrespective of their anatomical origin, the same type of copy number, expression profile, DNA methylation and miRNA changes. Over 94% of them had a mutation in one or more members of the WNT signaling pathway, predominantly in APC. However, there were some differences between tumors from the right colon and the remaining sites. Hypermethylation was more common in the right colon, and three quarters of hypermutated samples came from the same site, although not all of them had MSI (Figure 2). Why most of the hypermutated samples come from the right colon and why there are two classes of tumors at this site is not known. The origins of the colon from embryonic midgut and hindgut may provide an explanation. Since the survival of patients with high MSI cancers are better and these cancers have hypermutation, mutation rate may be a better prognostic indicator.  Whole exome sequencing and integrative analysis of genomic data provided further insights into the pathways that are dysregulated in CRC. We found that 93% of non-hypermutated and 97% of hypermutated cases had deregulated WNT signaling pathway. Novel findings included recurrent mutations in FAM123B, ARID1A and SOX9 and very high levels of overexpression of the WNT ligand receptor Frizzled 10. To our knowledge, SOX9 has not previously been described as frequently mutated in any human cancer. SOX9 is transcriptionally repressed by WNT signaling, and the SOX9 protein has been shown to facilitate β-catenin degradation38. ARID1A is frequently mutated in gynecological cancers and has been shown to suppress Myc transcription39. Activation of WNT signaling and inactivation of the TGF-β signaling pathway are known to result in activation of MYC. Our mutational and integrative analyses emphasize the critical role of MYC in CRC. We also compared our results with other large-scale analyses6 and find many similarities and few differences in mutated genes (Supplementary Table 3).  Our integrated analysis revealed a diverse set of changes in TCF/LEF encoding genes suggesting additional roles for TCF/LEF factors in CRC beyond being passive partners for β-catenin.  Our data suggest a number of therapeutic approaches to CRC. Included are WNT signaling inhibitors and small-molecule β-catenin inhibitors that are showing initial promise40–42. We find that several proteins in the RTK/RAS and PI3K pathways including IGF2, IGFR, ERBB2, ERBB3, MEK, AKT and mTOR could be targets for inhibition.  Our analyses show that non-hypermutated adenocarcinomas of the colon and rectum are not distinguishable at the genomic level. However, tumors from the right/ascending colon were more likely to be hypermethylated and to have elevated mutation rates than were other CRCs. As has been recognized previously, activation of the WNT signaling pathway and inactivation of the TGF-β signaling pathway, resulting in increased activity of MYC, are nearly ubiquitous events in CRC. Genomic aberrations frequently target the MAPK and PI3-K pathways but less frequently target receptor tyrosine kinases. In conclusion, the data presented here provide an unprecedented resource for understanding this deadly disease and identifying possibilities for treating it in a targeted way.  Go to: Methods Summary Tumor and normal samples were processed by either of two Biospecimen Core Resources (BCRs), and aliquots of purified nucleic acids were shipped to the genome characterization and sequencing centers (Supplementary Methods). The BCRs provided sample sets in several different batches. To assess any batch effects we examined the mRNA expression, miRNA expression and DNA methylation data sets using a combination of cluster analysis, enhanced principal component analysis, and analysis of variance (Supplementary Methods). Although some differences among batches were detected, we did not correct them computationally because the differences were generally modest and because some of them may reflect biological phenomena (Supplementary Methods).  We used Affymetrix SNP 6.0 microarrays to detect copy-number alterations. A subset of samples was subjected to low pass (2–5X) whole genome sequencing (Illumina HiSeq), in part for detection of SCNA and chromosomal translocations43,44. Gene expression profiles were generated using Agilent microarrays and RNA-Seq. DNA methylation data were obtained using Illumina Infinium (HumanMethylation27) arrays. DNA sequencing of coding regions was performed by exome capture followed by sequencing on the SOLiD or Illumina HiSeq platforms. Details of the analytical methods used are described in Supplementary Methods.  Postreplication DNA mismatch repair is essential for maintaining the integrity of genomic information in prokaryotes and eukaryotes. The first step in mismatch repair is the recognition of base-base mismatches and insertions/deletions by bacterial MutS or eukaryotic MSH2-MSH6. Crystal structures of both proteins bound to mismatch DNA reveal a similar molecular architecture but provide limited insight into the detailed molecular mechanism of long-range allostery involved in mismatch recognition and repair initiation. This study describes normal-mode calculations of MutS and MSH2-MSH6 with and without DNA. The results reveal similar protein flexibilities and suggest common dynamic and functional characteristics. A strongly correlated motion is present between the lever domain and ATPase domains, which suggests a pathway for long-range allostery from the N-terminal DNA binding domain to the C-terminal ATPase domains, as indicated by experimental studies. A detailed analysis of individual low-frequency modes of both MutS and MSH2-MSH6 shows changes in the DNA-binding domains coupled to the ATPase sites, which are interpreted in the context of experimental data to arrive at a complete molecular-level mismatch recognition cycle. Distinct conformational states are proposed for DNA scanning, mismatch recognition, repair initiation, and sliding along DNA after mismatch recognition. Hypotheses based on the results presented here form the basis for further experimental and computational studies.  Go to: Introduction DNA mismatch repair (MMR) pathways maintain the integrity of genomic DNA by eliminating errors incorporated during replication and recombination. The initial steps of DNA-mismatch recognition and repair initiation in the postreplication MMR pathway are mostly conserved from bacteria to humans with MutS in prokaryotes and MutS homologs (MSH) in eukaryotes recognizing defective DNA and initiating repair (1–3). A functional MSH protein leading to correct mismatch recognition and subsequent deletion is especially important in humans for the avoidance of cancer phenotypes (4).  Prokaryotic MutS is comprised of monomers with identical sequence, termed S1 and S2, although it forms a structural heterodimer when bound to DNA (5,6). MutS is known to recognize base-base mismatches and short base insertions or deletions, leading to their successful repair. In eukaryotes, at least seven variants of MSH have been identified. They form a number of heterodimers, of which MSH2-MSH6 corresponds most closely to MutS (with MSH2 corresponding to S2, and MSH6 corresponding to S1) (2). Like MutS, the MSH2-MSH6 complex also recognizes basepair mismatches and single base insertions or deletions with high efficiency, but does not efficiently recognize longer base insertions or deletions (1,2).  After MutS or MSH2-MSH6 is successfully associated with a mismatch, a complex is formed in the presence of ATP with MutL in prokaryotes (7) or MutL homologs (MLH) in eukaryotes (8) to promote downstream repair events. Crystal structures of prokaryotic MutS from Escherichia coli (E. coli) (6), Thermus aquaticus (5), and human MSH2-MSH6 (9) bound to different basepair mismatches or a single thymine insertion/deletion have become available. The structures all show the same architecture with two main functional sites at opposite ends of the dimer: a DNA-binding site and an ATPase site. As evidenced by the crystal structures, the clamp and DNA-binding domains (domains IV and I, respectively) from both chains (S1 and S2 of MutS or human MSH2-MSH6) encircle the mismatched DNA (Fig. 1). However, only the DNA-binding domain of one of the chains is in direct contact with the mismatch, giving rise to structural and functional asymmetry between the dimer moieties. Specific contacts with the mismatch base are made through a conserved “Phe-X-Glu” motif in the DNA-binding domain of chain S1 in MutS and MSH6. Insertion of this motif into the minor groove of the DNA is coincident with significant DNA bending (∼60°) and minor groove widening at and around the mismatch site compared to canonical DNA. The bent conformation of the DNA is further stabilized through nonspecific contacts from the clamp domain.  Figure 1 Figure 1 Crystal structure of MSH2-MSH6 in front (A) and sideways (B) orientation. Protein domains are indicated online in red (I, DNA binding), orange (II, connector), yellow (III, lever), green (IV, clamp) and blue (V, ATPase) and in shades of gray in print. ... The nucleotide-binding domains (NBDs; domain V) reside on the opposite end of the protein with the ATP-binding sites (ATPase sites) lying close to the dimerization interface. Biochemical studies have provided evidence for functional coupling between DNA scanning, mismatch recognition, repair initiation, and ATPase activity (10–13), which suggests allosteric signaling within the MutS or MSH dimers. Each MutS ATPase domain belongs to the ATP binding casette (ABC) superfamily (14) and is comprised of functionally important residues from both chains, as shown in Fig. 1 C (15). The NBD residing in each particular chain consists of Walker A and Walker B loops that are important for nucleotide phosphate binding and phosphate catalysis, respectively. Another loop containing a conserved phenylalanine residue (596 in MutS, 650 in MSH2, and 1108 in MSH6) stacks with the nucleotide adenine ring and the cavity is completed by the signature loop of the opposite monomer, which has been suggested to play an important role in catalysis. Several studies have suggested that the ATPase activities of the two chains are strongly correlated with each other and follow a sequential rather than simultaneous pattern of ATP hydrolysis (10). Moreover, both sites show intrinsic asymmetry in the ATPase activity, with nucleotide-binding affinities changing significantly for each ATPase site during the recognition cycle (10–13,16). In free enzyme or when bound to regular DNA, the chain that contacts the DNA mismatch (S1 or MSH6) has a higher affinity for ATP compared to the other chain, whereas chain S2 or MSH2 binds mostly ADP (13,16). It is further known that ATP hydrolysis occurs rapidly in S1/MSH6 when the protein is bound to regular DNA and that ADP release is the rate-limiting step (11). The ATPase site of the other chain has a much slower hydrolysis rate (13). These results highlight a differential behavior of the two ATPase sites when the protein is bound to regular DNA, as depicted schematically in Fig. 2. During scanning of regular DNA, the NBD of chain S1/MSH6 binds ATP followed by fast hydrolysis to ADP. However, since exchange of ADP for ATP does not occur as rapidly as hydrolysis, ADP will be bound to this site for the majority of the time. At the same time, ADP is also bound predominantly to the other NBD of chain S2/MSH2.  Figure 2 Figure 2 The dynamic behaviors of the ATPase site in both chains are represented along an arbitrary horizontal time axis. Alterations among three possible nucleotide-binding states (ADP/ATP/free) of the NBD are shown along the vertical axis with the help of curves ... Experimental data suggest that mismatch binding promotes the exchange of ADP for ATP while stalling ATP hydrolysis of S1/MSH6 (11,12). The resulting prolonged ATP-bound state at S1/MSH6 “authorizes” recognition of a mismatch by the DNA binding domain, whereas ATP is readily hydrolyzed when the DNA binding domain is bound to regular DNA (17). Furthermore, stable ATP binding by S1/MSH6 ultimately leads to reduced ADP-binding affinity in the ATPase site of S2/MSH2. This presumably enhances the ATP-binding affinity of S2/MSH2 (13). The dual ATP-bound state is believed to trigger a conformational change to a sliding clamp conformation whereby the mismatch is released by the DNA-binding domain and rebinding of mismatched DNA is inhibited (12,13). Of interest, a recent single-molecule study on MSH2-MSH6 demonstrated that the sliding motion along DNA after mismatch recognition is independent of ATP hydrolysis (18).  Although there is a general understanding of the long-range allostery of MutS and its homologs involved in recognition and repair initiation, the molecular-level events leading to the functional correlation between N-terminal DNA mismatch recognition and C-terminal nucleotide binding and hydrolysis have remained elusive. Further advances in this respect have been hindered by the fact that the available crystal structures show only the mismatch-bound state and do not provide information about the different nucleotide-bound combinations in the two ATPase domains. Until now, the complex interplay between functional states of the two ATPase- and DNA-binding sites has mostly been elucidated by biochemical kinetic studies that failed to provide a molecular-level understanding of the process. Experiments may continue to reveal additional information for different functional states; however, conformational sampling of proteins can also be studied by theoretical means. Molecular-dynamics simulations that often offer insights in this regard are not easily applicable to MutS because of the long timescales of the mismatch recognition process and the large system size of the MutS-DNA complex. Normal-mode analysis (NMA) is an alternative strategy for studying large-scale conformational changes in biomolecules. NMA relies on a harmonic approximation of the potential energy surface around a minimum energy structure, and the resulting lowest-frequency dynamic modes often resemble biologically relevant functional motions (19,20). Here, we applied NMA to study the conformational dynamics of MutS and its eukaryotic homolog MSH2-MSH6. The results suggest a new molecular-level understanding of the long-range allosteric pathway in the functional interplay among DNA mismatch recognition, nucleotide-binding activity, and repair initiation. A structural characterization of distinct conformational states, along with the elucidation of a complete functional cycle, offers possible avenues for validating the proposed cycle through experiments.  Go to: Materials and Methods Normal-mode (NM) calculations were performed on E. coli MutS and its human homolog, MSH2-MSH6, in the absence of any bound nucleotides. Calculations were performed on each protein in the presence or absence of DNA, resulting in a total of four sets of NM calculations. These are referred to as MSH-DNA, MSH-free, MutS-DNA, and MutS-free for MSH2-MSH6 with DNA, MSH2-MSH6 without DNA, MutS with DNA, and MutS without DNA, respectively. Initial structures of E. coli and human protein were obtained from Protein Database (PDB) IDs 1E3M (6) and 2O8B (9), respectively, and missing loops were constructed using MODELLER (21). The structures were then extensively energy minimized using the CHARMM22/CMAP force field (22) and distance-dependent dielectric (ɛ = 4). The root mean-square deviations (RMSDs) of the minimized structures with respect to the crystal structures were 2.08Å, 2.42Å, 1.28Å, and 2.06Å for MSH-DNA, MSH-free, MutS-DNA, and MutS-free, respectively. Low RMSD values indicate that extensive minimization in the absence of explicit water or DNA does not lead to significant structural deviations from the crystal structure. NMs were calculated by means of the block-NM approach using the VIBRAN module in CHARMM (23,24), version c33a2, and with the same force field as used for minimization. Only low-frequency modes were analyzed in both proteins because they are the most relevant for describing functional motions involving the entire complex. To calculate the similarity between individual modes among MutS and MSH2-MSH6, we defined the overlap index for each pair of modes (i,j) as |(∑kSik⋅Hjk)/k|, where k is the number of aligned residues of MutS and MSH, Sik is the kth component unit vector of the ith mode of MutS, and Hik is the kth component unit vector of the jth mode of MSH. Each dot product contributing to the sum is between unit vectors and can possess a maximum value of 1 for residue pairs moving in exactly the same direction, or a value of −1 for residue pairs moving in exactly the opposite direction. The value of the overlap index can thus reach a maximum value of 1 for an ideal case in which all aligned residues of two proteins are moving in exactly the same or opposite direction. The sequence alignment between MSH2-MSH6 and MutS was taken from previous work (9). Molecular graphics were generated using PyMOL (25).  Go to: Results and Discussion NM calculations were carried out to explore the possible conformational dynamics of MutS and MSH2-MSH6 from the perspective of the crystal structures. Apart from conducting NMA calculations on both proteins with bound DNA, we also considered proteins without DNA to allow dynamics beyond the DNA mismatch bound form. Results from the analysis of MutS and MSH2-MSH6 in the presence and absence of DNA are discussed below. Data from four different sets of NM calculations are referred to as MSH-DNA, MSH-free, MutS-DNA, and MutS-free for MSH2-MSH6 with DNA, MSH2-MSH6 without DNA, MutS with DNA, and MutS without DNA, respectively.  Flexibility of MutS and MSH2-MSH6 from NMs and x-ray data  Root mean-square fluctuations (RMSFs) provide information about inherent protein flexibility. They can be deduced from experimental B-factors or can be calculated from NMs (26). The results in Fig. 3, A–D, show that the RMSFs calculated from experimental B-factors are uniformly high due to the limited resolution of the MSH2-MSH6 crystal structure (2.75 Å) and do not provide significant information about relative domain fluctuations. RMSFs from B-factors of the MutS crystal structure (with a resolution of 2.2 Å) are still high but indicate increased flexibility in MutS domains I and IV, and parts of III, in particular for chain S2. In contrast, RMSFs calculated from the first 10 NMs show significant differences in the domain movements. MutS-free and MSH-free exhibit large flexibility in the DNA binding and clamp domains (I and IV) and a lesser degree of flexibility between the lever domains (III). In contrast, the ATPase domains (V) show comparably low structural fluctuations. Mode calculations for MutS-DNA and MSH-DNA provide qualitatively similar results, but with damped flexibility in the clamp domains and in the DNA-binding domains of chain MSH6 and MutS S1. Fig. 4, A–D, show both proteins colored according to the B-factors calculated from NM RMSF values.  Figure 3 Figure 3 RMSF of Cα atoms as a function of residue number calculated from the first 10 NMs (online: red, without DNA; black, with DNA) and from crystallographic B-factors according to RMSFXray=3B/8π2−−−−−−√ (blue, thick dots) for MSH6 (A), MSH2 (B), MutS ... Figure 4 Figure 4 Protein backbone showing thermal fluctuations color-coded by B-factors calculated from RMSF of the first 10 modes for MSH-free (A), MSH-DNA (B), MutS-free (C), and MutS-DNA (D). The color scale for B-factors is provided at the end of the figure. The NMA-based dynamics of MutS and MSH2-MSH6 are remarkably similar between chains, as well as between the prokaryotic and eukaryotic enzymes. MSH6 and domain I of MSH2 appear to be slightly more rigid compared to MutS, which may relate to the functional specialization of MSH2-MSH6. One may speculate that MutS requires increased flexibility to recognize both mismatches and longer insertions/deletions, in contrast to MSH2/MSH6, which recognizes only mismatches and single base insertions/deletions. Although an absolute comparison of RMSF values from a small number of NMs between proteins of different sizes may be problematic, very similar results were obtained when motions of the first 100 modes were accumulated (data not shown).  Furthermore, it appears that MSH2 and MutS S2 are slightly more flexible than MSH6 and MutS S1, respectively, with the exception of the clamp domain for MSH-DNA and MutS-DNA systems. The increased flexibility in domain I of MSH2 or MutS S2 and the decreased flexibility of the clamp domain compared to the other chain correspond to the structural asymmetry of MutS and MSH2-MSH6. The increased flexibility of domain I of MSH2/MutS S2 probably results from the fact that they do not make considerable contacts with the DNA, whereas extensive DNA contacts of the clamp domains of MSH2/MutS S2 compared to the other chain accounts for its decreased flexibility. It was observed that the clamp domain of MSH2 and MutS S2 makes 83 and 86 atomic contacts with the DNA, respectively, whereas that of MSH6 and MutS S1 makes only 62 and 42 contacts, respectively. It was further observed that the clamp and lever domains of MutS S1 are more flexible compared to those of MSH6, which again is a result of fewer atomic contacts made by the clamp of S1 with the DNA compared to MSH6. The number of atomic contacts was calculated by considering protein heavy atoms around 5Å of the DNA in the minimized structure of both proteins. Finally, Fig. 4 highlights that the clamp and lever domains of MSH2 in MSH-DNA are slightly more flexible than the corresponding domains in MutS S2 of MutS-DNA. These differences are probably the result of the proteins being bound to DNA segments of varying lengths. MutS-DNA has a longer DNA (3 basepair steps more than MSH-DNA), which topologically constrains the mobility of MutS S2, giving rise to a more rigid S2 clamp compared to that of MSH2. The portion of the lever domain of S2 that is tightly connected to the clamp also undergoes some degree of rigidification. The rigidification of MutS S2 may be more close to reality, as DNA undergoing repair in the cell is much longer than that observed in the crystal structures. It should also be mentioned that substantially longer DNA can alter the extent of flexibility observed in the clamp of the other chain, namely, S1 and MSH6. Hence, difference in the flexibility of clamp and lever due to the presence of a much larger DNA cannot be directly inferred from these studies using fragmented DNA. The only observed fact is that an overall decreased flexibility of the clamps and levers will result in both chains when compared to DNA-free systems.  Finally, it was observed that the RMSF for protein with DNA spikes at residues 1275–1281 in MSH6 and residues 663–666 in MutS S2. This is likely a manifestation of the tip effect (27) and is considered physically meaningless.  Correlated motions in MutS and MSH2-MSH6  Covariance plots averaged over the 10 lowest NMA modes were calculated to examine correlated motions in all four systems under investigation. The results shown in Fig. 5 indicate similar overall correlations in MutS and MSH2-MSH6 in the absence and presence of DNA. Furthermore, both chains of MutS and MSH2-MSH6 show similar average correlation patterns, with only minor variations, despite the structural asymmetry of the complex. Common to all chains are correlations within each domain, reflecting rigid-body domain motions, such as correlations between adjacent domains II (connector domain) and III (lever domain), between III and V (ATPase domain), and between I (DNA binding domain) and II. Whereas correlations within the same subunit are generally positive, correlations between dimer moieties are mostly negative, with the exception of a strong positive correlation between the two ATPase domains and the two clamp domains as a result of dimerization.  Figure 5 Figure 5 Average covariance from the first 10 modes in MSH2-MSH6 (A, upper triangle: MSH-free; lower triangle: MSH-DNA) and MutS (B, upper triangle: MutS-free; lower triangle: MutS-DNA). Protein domains in both chains are indicated by colored bars following ... The plots indicate a high positive correlation between the lever domains and parts of the ATPase domains immediately adjacent to the lever, including the ATPase-binding sites. Experiments suggest the presence of long-range allostery between the N-terminal DNA-binding domain and the C-terminal ATPase domains; however, a clear understanding of the allosteric pathway is lacking. Strong correlations between the ATPase sites and lever domains highlight the propagation of signals within the two functional sites via the levers. Furthermore, the DNA-binding domain in MSH6 and MutS S1 has a strong negative correlation with the ATPase domain in MSH2 and MutS S2, respectively, in particular for MSH-DNA and MutS-DNA, again suggesting conserved domain motions that are important for allostery.  Correspondence between MutS and MSH2-MSH6 modes  The analysis of RMSFs and motional correlations indicates that MutS and MSH2-MSH6 exhibit similar dynamic characteristics in both the presence and absence of DNA. Furthermore, the correlation analyses from the first 10 modes suggests dynamic coupling between DNA binding and ATPase activity. To explore this point in more detail, the 10 lowest-frequency modes were individually compared between different states of the same protein, i.e., between MSH-free and MSH-DNA or between MutS-free and MutS-DNA. The same comparison was also performed between different proteins, i.e., MSH-free and MutS-free and MSH-DNA and MutS-DNA. Table 1 shows the overlap indices calculated between any pair of modes from four different systems as described in Materials and Methods. An overlap index value of 1.0 means that atoms move in identical directions in the two modes that are compared, and a value of 0.0 means that motions are entirely orthogonal or that atom motions have zero amplitude. Although a value of 1.0 or close to it is unlikely even for very similar structures, visual inspection of the MutS and MSH modes indicates that the motions are qualitatively similar when overlap indices are at 0.6 and above and, to a lesser but still substantial extent, when values are between 0.5 and 0.6, especially when MutS is compared with MSH. Relatively low overlap indices despite visually similar motions are due to uncertainties in the alignment between the two proteins, with a sequence identity of only 21% and 24% for MSH2 and MSH6 (9); differences in structure; and significant overall flexibility due to the multidomain nature of both MutS and MSH.  Table 1 Table 1 Overlap index for a pair of modes, each from two different sets Table 1 shows that the highest degree of overlap on a mode-by-mode basis exists between MSH-free and MSH-DNA, and also between MSH-free and MutS-free systems. There is a lesser degree of one-to-one correspondence between MutS-free and MutS-DNA, and also between MSH-DNA and MutS-DNA, with individual modes being reordered more significantly according to frequencies in these systems. High one-to-one overlap is found between modes 1, 2, 3, 4 of MSH-free and modes 1, 5, 3, 4 of MutS-free; between modes 1, 3, 4, 9, 10 of MSH-DNA and modes 2, 5, 4, 9, 10 of MutS-DNA; between modes 1, 2, 4, 5, 8, 9 of MSH-DNA and modes 2, 3, 4, 5 and 9, 10, 7 of MSH-free; and between modes 1, 2, 3, 5 of MutS-DNA and modes 2, 5, 3 and 9, 7 of MutS-free. It is apparent that many modes do not match on a one-to-one basis, but share common features with multiple modes (e.g., mode 2 of MSH-free matches modes 2, 5, and 8 of MutS-free). It is known from previous studies that the complex domain motions that are responsible for altered functional states in a large protein are often better represented as a combination of low-frequency modes. Our studies also suggest that the low-frequency modes of both MSH and MutS exhibit almost similar domain flexibility, whereas specific protein dynamics are often seen to occur as a combination of multiple modes showing different degrees of mode mixing in both proteins. The degree of mode mixing observed in this study will likely change as a result of using different force fields or coarse-grained models, but low-frequency NM space will likely be conserved in all NM analyses, provided the starting structure remains the same. Thus, the main aim of this study is to highlight the conserved nature of domain motions in both proteins, rather than to highlight any specific mode or modes responsible for the protein function.  The presence of DNA alters the structural flexibility to some extent, as evidenced by the differences between modes in the presence and absence of DNA. For example, mode 1 is present in MSH-free and MutS-free but not in MSH-DNA or MutS-DNA. As described in more detail below, the mode involves large motions of the clamp domains that are not possible in the presence of DNA. Visual inspection further reveals that altered motions of the clamp and DNA-binding domains for structures in the presence and absence of DNA is a major factor in reduced mode overlap indices between the two protein systems, despite otherwise similar overall motion. Of interest, MSH modes are much more conserved in MSH-free and MSH-DNA systems than in the two MutS systems. This suggests that protein flexibility is altered more in MutS than in MSH2-MSH6 through specific DNA interactions, especially near the DNA-binding domain and clamps. This further reflects a more rigid overall structure in MSH2-MSH6 that is optimized to interact with mismatched DNA, whereas MutS requires more structural flexibility to interact with both mismatched DNA and significantly distorted DNA structures with insertions or deletions.  In this study, we focus more on modes from MSH-free and MutS-free because they are more likely to indicate motions from the known mismatch-bound crystallographic structures toward alternate states during DNA scanning and MMR. A comparison of the modes between MSH-free and MutS-free indicates that modes 1, 3, 4, and 9 from both complexes significantly overlap and may be considered equivalent. Modes 2 and 5 overlap significantly, suggesting that these modes are simply reordered with respect to their frequencies. However, there is also overlap between mode 2 of both complexes, suggesting common features in both modes. Otherwise, there is significant mode overlap along the diagonal for modes 7–9 and additional limited off-diagonal overlap for modes 6–10. Overall, mode overlaps between MSH-DNA and MutS-DNA are lower, but high overlap indices are again limited mostly to diagonal or nearby off-diagonal elements, with modes 1, 2, 3, 4, 5, 6, 9, and 10 of MSH-DNA corresponding to modes 1 and 2, 2, 5, 4, 6, 6, 9, and 10 of MutS-DNA.  Low-frequency modes in MSH2/MSH6  Our analyses indicate that the dynamic characteristics are largely conserved between MutS and MSH2-MSH6 in both the presence and absence of DNA. This is not surprising, but it is also not trivial given the structural differences between the eukaryotic and prokaryotic enzymes and the slightly different biological functions. In the following, we describe the lowest-frequency modes in more detail, with a focus on the modes of MSH-free.  The protein motion during each of the first five modes of MSH-free is shown in Fig. 6. Close-up views of the ATPase domains of selected modes are shown in Fig. 7. Modes 6–10 of MSH-free and all 10 modes of MutS-free and MutS-DNA, as well as those for MSH-DNA, are shown in Figs. S1–S5 in the Supporting Material. Movies for mode 1 (Movies S1a and S1b), mode 2 (Movies S2a and S2b), mode 3 (Movies S3a and S3b), mode 4 (Movies S4a and S4b), and mode 5 (Movies S5a and S5b) of MSH-free are also provided. Initial visual inspection suggests the following general conclusions about the nature of domain motions in both MutS and MSH2-MSH6: 1), Most of the modes show an overall breathing motion of the DNA-binding cavity involving the clamp and DNA-binding domains. The parts of the clamp domains that are directly bound to the DNA backbone always show damped motion in proteins with DNA, although movements of other parts of the DNA-binding cavity show a similar kind of breathing motion. Such an opening/closing motion of the DNA-binding cavity corresponds to conformational transitions between a mismatch-bound state and scanning/sliding conformations where the interaction with DNA is presumed to be weaker. 2), Many modes show a correlation between opening/closing of the DNA-binding cavity and alterations in the ATPase domain, in particular the nucleotide-binding cleft. This finding establishes that both MutS and MSH2-MSH6 are capable of allosteric communication between DNA-binding and ATPase activity. The correlation between motions of the DNA-binding domains and the ATPase domains varies because it may involve the MSH6, MSH2, or both ATPase domains in an alternating fashion. 3), A mode that affects the nucleotide-binding cleft in both ATPase domains in the same manner and at the same time is not observed in any of the four cases studied. This finding agrees with the experimental evidence that ATPase activity in MSH2-MSH6 involves the two domains in a sequential rather than a simultaneous fashion (10).  Figure 6 Figure 6 Mode motions of MSH-free projected onto the minimized crystal structure for modes 1 (A), 2 (B), 3 (C), 4 (D), and 5 (E). Motions are indicated by colored arrows (online) in the direction of the mode vectors for every sixth residue. Motions involving the ... Figure 7 Figure 7 Close-up views of the motions in the NBD of MSH-free during modes 2 (A), 3 (B), and 5 (C). Arrows are placed on each Cα atom in every three consecutive residues, and only displacements of more than 1Å are shown. The two chains and bound ... The individual modes are described in detail below:  Mode 1 involves a wagging motion of the clamp domain along the direction of the DNA. The rotating motion around the core, which is apparent in the rest of the enzyme, results from a fixed center of mass. If the protein is aligned at domains I, II, III, and V, only the clamp domain IV moves in this mode. Mode 1 involves both chains to the same extent. The exact functional role of this mode is unclear, but it may be related to the translocation of MSH2-MSH6 along DNA in the absence of mismatch when the clamps do not establish strong contacts with the DNA backbone. This mode is absent in both proteins bound to DNA mismatch, presumably due to residue contacts with the bent DNA.  Mode 2 consists of a partial opening/closing motion of the DNA-binding site that is less pronounced than in some of the other modes. The unique aspect of this mode is the alternating opening/closing of the nucleotide-binding clefts between the MSH2 and MSH6 ATPase domains (Fig. 7). It appears likely that this mode is involved in coupling MSH6 and MSH2 ATPase activity in a sequential fashion. As mentioned above, mode 2 in MSH-free has high overlap with mode 5 of MutS-free. However, this is due to similar motions in the DNA-binding, clamp, and core domains. The alternating opening/closing of the two nucleotide-binding sites is not present in mode 5 of MutS-free, but is seen instead in mode 2 of MutS-free. An alternating ATPase movement that correlates similarly to motions in the DNA-binding cavity is also observed in mode 1 of MutS-DNA and MSH-DNA, suggesting that the interdomain correlation is conserved in all systems.  Mode 3 couples opening of the DNA-binding site with closing of the nucleotide-binding cleft in MSH2. The opening of the DNA-binding site is achieved by the movement of the clamp domains away from the DNA, as well as the movement of the DNA-binding domain in MSH6 out of the plane of the MSH2-MSH6 complex. Relative to the DNA, this motion moves domain I out of the DNA groove rather than along its helical axis. In the open form of this mode, most DNA contacts of MSH6 near the mismatch site are lost and the DNA can essentially slide freely relative to MSH2-MSH6. This mode is highly conserved in all other systems of MutS and MSH, and is thus expected to play an important role in the protein's functional cycle. An almost identical mode is observed for modes 2 and 3 for MSH-DNA and MutS-DNA. We note, though, that the overlap index between the two modes in MSH-DNA and MutS-DNA is small due to altered clamp movements in MutS-DNA, but otherwise they show similar domain motions.  Mode 4 consists mainly of a sideways motion of the clamp and part of the lever domains toward either the MSH2 or MSH6 side of the enzyme. This mode is asymmetric with respect to the overall complex. A symmetric version of this mode would result in clamp domain separation and lead to an open dimer where the clamp domains are far away from each other, as proposed for the DNA-free complex from small-angle x-ray scattering (28). The symmetric mode is not observed, presumably due to limitations of the harmonic approximation in NMA.  Mode 5 involves closing of the DNA-binding cavity that is coupled with opening of the nucleotide-binding cleft in MSH6. The closing of the DNA-binding cavity is achieved primarily by the motion of the clamp domains directly toward the DNA. A similar overall motion is also found in mode 2 of MutS-free, although the coupling between opening and closing of the DNA-binding cavity with changes in the ATPase domain of MutS S1 is more pronounced in mode 6 of MutS-free. It is likely that MutS-free achieves a motion equivalent to the MSH-free mode 5 through a combination of modes 2 and 6. A similar correlated motion between the DNA-binding cavity and ATPase domains is further observed in mode 5 of MSH-DNA and mode 4 of MutS-DNA.  Functional cycle of MSH2-MSH6 and MutS from NMs  The crystal structures of MutS and the MSH2-MSH6 complex show only the mismatch-bound conformation. It is clear, however, that other functional states are involved during scanning of regular DNA, authorization of MMR, and sliding of the enzyme along DNA during and immediately after repair before DNA scanning is resumed. On the molecular level, these different states are likely reflected in altered conformations of MSH2-MSH6 and MutS. X-ray crystallographic approaches have not identified alternate states of MSH2-MSH6, but there is evidence of alternating ATP- and ADP-bound states from small-angle x-ray scattering (28), where ATP binding has resulted in more compact protein conformations. The NMA presented here offers for the first time, to our knowledge, insights into the functional dynamics of MSH2-MSH6 and MutS beyond the known DNA-mismatch bound crystal structures. By combining the conserved low-frequency modes in a sequential fashion, it is possible to propose, for the first time, a complete functional cycle of MSH2-MSH6 and MutS that is in full agreement with experimental observations. The proposed molecular-level picture of the cycle is illustrated in Fig. 8 and described in detail below.  Figure 8 Figure 8 Schematic diagram representing distinct conformational states during the functional cycle of MSH2-MSH6 or MutS. DNA binding  The functional cycle of MSH2-MSH6 and MutS begins with binding to newly replicated DNA. Experimental data suggest that DNA-free MutS is present in an open form. Upon association with DNA, the clamp domains are presumed to close. The asymmetric mode 4 indicates how the clamp domains might separate, starting from the DNA-bound form, without significantly affecting the structure of the rest of the enzyme.  DNA scanning and mismatch recognition  Once MSH2-MSH6 or MutS is bound to DNA, it will begin scanning for base mismatches. According to single-molecule experiments, MSH2-MSH6 moves along regular DNA via one-dimensional diffusion (18), whereas DNA-binding kinetics indicate that the protein is not bound strongly to DNA in the absence of a mismatch (1,29,30). In contrast, MSH2-MSH6 and MutS interact closely with mismatched DNA in a highly bent form, as evidenced by the crystal structures (5,6,9). The formation of highly bent DNA is greatly facilitated by the presence of base mismatches or base insertions/deletions (31) and is believed to be the main feature by which mismatch DNA basepairing is recognized (1). The transition from scanning to mismatch recognition is therefore expected to involve a significant change in the DNA-binding domain from a relaxed conformation with relatively weak protein-DNA interactions to a tightened conformation in which the enzyme holds on to highly bent DNA. The opening/closing motion of the DNA-binding cavity in mode 5 of MSH2-MSH6 describes such a transition in molecular detail.  The transition from DNA scanning to mismatch recognition is coupled to the fast exchange of ADP to ATP and subsequent stalling of ATP hydrolysis in MSH6, according to kinetic experiments (10–13). Mode 5 couples closing of the DNA-binding cavity to opening of the MSH6 or MutS S1 nucleotide-binding cleft, and vice versa. The nucleotide-binding cleft is sandwiched between the Walker A motif and a loop, which acts as a flap over the adenine moiety. This loop contains a conserved Phe residue (Phe596 in MutS, Phe650 in MSH2, and Phe1108 in MSH6) that stacks with the adenine ring in all available crystal structures. Previous studies of ATP binding in some ATPases revealed that binding often induces tightening of the site that is required for ATP hydrolysis, as suggested by an increase in the hydrophobicity of the binding pocket (32) or closing of specific loops in the presence of the nucleotide, resulting in a tightened cavity (33). We hypothesize that an open nucleotide-binding cleft in the MSH2-MSH6 and MutS ATPase domains encourages ATP binding but inhibits hydrolysis. In contrast, closing of the ATPase cavity predominantly involves movement of the loop bound to the adenine ring toward the catalytic center (Walker B motif), thereby ensuring successful ATP catalysis. Mode 5 therefore provides a molecular-level picture of how mismatch recognition through deformation of DNA at the mismatch site might be coupled to the experimentally observed changes in MSH6 ATP activity.  In most of the crystal structures of MSH2-MSH6 and MutS, ADP or a nonhydrolyzable ATP analog is present in both or any one of the chains, although only one MutS structure with bound ATP on both chains has been reported so far (15). The difficulties of observing the protein with stable ATP bound MSH6/S1 may be attributed to crystal packing that does not allow the formation of ATPase domains that are truly catalytically inactive (15). Thus, the crystal structure with ATP at S1 may not be fully representative of the true nonhydrolyzable state of the protein; rather, it may represent a different trapped intermediate state.  Initiation of repair  The next step after base mismatch recognition is initiation of the repair process. This involves binding of MutL/MLH to MutS/MSH2-MSH6 (1–3), which then signals further downstream events. Furthermore, kinetic studies indicate that ADP exchanges for ATP in the MSH2 ATPase domain subsequent to ATP binding in the MSH6 ATPase domain (13). The sequential coupling of ATP binding to the two ATPase domains mirrors alternating ATP hydrolysis activity in other dimerized ATPase domains as in ABC transporters (34), and can be understood in terms of the alternating opening/closing of the nucleotide-binding clefts seen in mode 2. We hypothesize that the initially very open ATP-binding site in MSH6 after mismatch recognition partially closes upon ATP binding, which in turn leads to opening of the MSH2 ATP-binding site according to mode 2 and subsequent exchange of ADP for ATP in MSH2. Mode 2 also involves structural rearrangement outside the ATPase domain, indicating that the enzyme assumes a distinct conformation at this step of the functional cycle, possibly to facilitate MutL/MLH binding.  Repair and mismatch release  Recent experiments suggest that after the initiation of repair, MSH2-MSH6 and MutS form a mobile clamp state that slides along the DNA in search of downstream repair proteins (12,13,18,35,36). A transition from the mismatch-bound state to a sliding conformation requires reopening of the DNA-binding cavity to a form that still holds the DNA but is not competent to rebind mismatched DNA (12). Moreover, this sliding activity is not powered by ATP hydrolysis. This transition remains to be elucidated by experimental studies. We propose that the most conserved mode in both proteins, i.e., mode 3, describes the molecular events involved in the formation of sliding clamp conformation. In mode 3, the DNA-binding cavity is opened by the release of the clamps from the DNA coupled to a large motion of the DNA-binding domain perpendicular to the DNA helix. As a result, intimate interactions with the mismatch through the DNA groove become impossible. This is particularly true for interactions involving the highly conserved Phe-X-Glu motif, which is known to interact specifically at the mismatch site (5,6,9,15). The protein is capable of sliding along the DNA in this state. The release of DNA mismatch binding and sliding according to mode 3 is coupled to a tightening of the MSH2 ATP-binding site, which would facilitate eventual ATP hydrolysis in MSH2 and allow recovery of the DNA scanning mode.  Although specific NMs have been mentioned in descriptions of molecular events during scanning, mismatch binding, and sliding clamp formation, it is likely that opening and closing of the DNA-binding cavity actually occurs as a result of multiple low-frequency modes. This is even more likely because almost all of the low-frequency modes studied, except mode 1 in MSH-free and MutS-free, exhibit some kind of breathing motion of the DNA-binding cavity that involves different domain motions, such as those of the clamps, levers, and DNA-binding domain. The specific modes used to describe the conformational changes in the functional cycle only show the necessary synchronization between the opening/closing of the DNA-binding cavity and the ATPase cleft, and thus are used to describe the experimentally observed allosteric effects.  Validation through experiments and further simulation  The results from the NM calculations presented here allow us to make a number of predictions about the functional dynamics of MutS and MSH2-MSH6 and the existence of additional functional states that have not been characterized on a molecular level to date. In particular, we propose molecular-level details of long-range allosteric coupling between the N-terminal DNA-binding domains and the C-terminal ATPase sites, as well as coupling between the two adjacent ATPase sites, which are known to exhibit a sequential pattern of action. In addition, the results presented here provide an atomic level characterization of distinct states in the functional cycle of MutS and MSH2-MSH6. A more open DNA scanning conformation is proposed and a sliding clamp state is predicted whereby the DNA-binding domain is rotated out of the enzyme to result in structures that are significantly different from the crystal structure. These findings should stimulate further experimental and computational studies to validate the predictions made here. In particular, structural experiments could probe the nature of the DNA scanning and sliding conformations based on the predictions presented here, and biochemical studies could test mutations that would disrupt the proposed domain movements. Furthermore, the proposed structures for alternate functional states could be subjected to more extensive computational studies to examine their stability and transitions between those states.  Go to: Conclusions Results from NM calculations of MSH2-MSH6 and MutS were used to develop a molecular-level picture of distinct conformational states involved in their functional cycles. A comparison of the modes between MSH2-MSH6 and MutS reveals striking similarities, indicating that the two enzymes are not just structurally but also dynamically and functionally equivalent on the molecular level. The most important result indicates the presence of a strong motional correlation between the ATPase domains and the lever domains in all low-frequency modes analyzed, whereas individual modes highlight the specific nature of the correlation between the N-terminal DNA-binding domains and the ATPase domains. This indicates that both MutS and MSH2-MSH6 are structurally capable of establishing long-range allostery during their functional cycle. Based on a detailed analysis of the lowest-frequency modes in the context of the available experimental data, a detailed mechanism is proposed that involves DNA scanning, mismatch recognition, repair initiation, and sliding of MSH2-MSH6/MutS along DNA before scanning is resumed.  NM calculations can provide an approximate view of biologically relevant dynamics in biomolecules, but are limited by the theoretical nature of the methodology. The ideas presented here suggest a number of experiments that could validate and extend the proposed mechanism of DNA mismatch recognition by MSH2-MSH6 and MutS. Furthermore, the NM results can serve as starting points for additional computational studies to investigate the proposed functional states and transitions between them in more detail.\""
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "txt_train[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
